challenge-find-the-largest-number/script.js

```diff
@@ -1,7 +1,36 @@
-function calculateLargestNumber(arrayOfNumbers) {
-  // Write your code in here
+// First solution with using "FOR LOOP" method.
+// function calculateLargestNumber(arrayOfNumbers) {
+//   let largest = 0;
```

Comment:
You accepted challenge, congrats. You passed for these tests but if an array has all negative number it doesn't work so you need to start 'largest' with an element of array.